backend/fregepoc/repositories/exceptions.py

```diff
@@ -0,0 +1,8 @@
+class DownloadDirectoryFullException(Exception):
+    def __init__(self, message: str):
+        super(DownloadDirectoryFullException, self).__init__(message)
```

Comment:
Maybe it is worth sticking to a new `super()` invocations - as it is done in the class below.